tssh/agent_windows.go

```diff
@@ -0,0 +1,51 @@
+package tssh
+
+import (
+	"os"
+
+	"github.com/natefinch/npipe"
+	"golang.org/x/crypto/ssh/agent"
+)
+
+// AGENT_PIPE_ID is the default pipe id for openssh ssh-agent on windows.
+const AGENT_PIPE_ID = `\\.\pipe\openssh-ssh-agent`
+
+func getAgentSigners() []*sshSigner {
+	pipeId := AGENT_PIPE_ID
+	// get enviroment override
+	if socket := os.Getenv("SSH_AUTH_SOCK"); socket != "" {
+		pipeId = AGENT_PIPE_ID
```

Comment:
Is it override nothing?

Reply:
ssh-agent could use a different pipe id, which would be passed through SSH_AUTH_SOCK

Reply:
Shouldn’t it be `pathId = socket`?

Reply:
Right, I made a mistake. Thank you.